Modules/timemodule.c

```diff
@@ -740,8 +740,7 @@ checktm(struct tm* buf)
 }
 
 #ifdef MS_WINDOWS
-   /* wcsftime() doesn't format correctly time zones, see issue #10653 */
-#  undef HAVE_WCSFTIME
+#  define HAVE_WCSFTIME
```

Comment:
There are compiler warnings: 'HAVE_WCSFTIME': macro redefinition [D:\a\cpython\cpython\PCbuild\_freeze_module.vcxproj]

Reply:
Oh, I did not find where it was defined in C.